src/FeedbackBot/AppSettings.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace FeedbackBot.Controllers
```

Comment:
@cszhu probably want to move it out of the controllers namespace, maybe put it in the root or models folder.  no big deal for now